test/rackup/realistic_response.ru

```diff
@@ -6,6 +6,6 @@ long_header_hash = {}
   long_header_hash["X-My-Header-#{i}"] = SecureRandom.hex(25)
 end
 
-response = SecureRandom.hex(100_000) # A 100kb document
+response = SecureRandom.hex(50_000) # A 100kb document
```

Comment:
What was the reason for the change here? should update the comment at least

Reply:
```
ruby -rsecurerandom -e "puts SecureRandom.hex(100_000).length"
```
outputs 200_000, or `SecureRandom.hex(b_len)` creates a byte sequence of `b_len` bytes, and then converts it to a hex string.  Each byte turns into 2 characters, so...